App/Containers/TextilePhotos.js

```diff
@@ -187,7 +188,11 @@ const mapStateToProps = (state, ownProps) => {
     thread = state.threads.threads.find(thread => thread.id === threadId)
   }
 
-  const threadName = thread ? thread.name : undefined
+  // I saw a really weird state where thread was all undefined....
+  // seems like we should show a loading state if that ever happens.
+  // at the very least i put the user on the default screen instead of a
+  // blank Thread screen
+  const threadName = thread ? thread.name : 'default'
```

Comment:
@asutula potential change here of significance. i can't see when a thread.name wouldn't exist but in a broken state i at least try to put them in the wallet view instead of a thread view (with broken back button etc)

Reply:
Well there's a period of time when `thread` is `undefined`. This happens quickly while the thread data is being async fetched from the node. Yea, we should display a loading indicator or something. 

I'm not sure forcing a thread name of `default` really fixes anything, it may even be inconsistent with the `threadId` at the same time.